messages/en.json

```
@@ -382,5 +383,235 @@
+    "FAQ_SETUP_TIME_ANSWER": "With our quick start guide, you can have Ever Works running in under 10 minutes.",
+    "FAQ_CODING_EXPERIENCE": "Do I need coding experience?",
+    "FAQ_CODING_EXPERIENCE_ANSWER": "Basic knowledge of web development helps, but our detailed guides make it accessible to beginners.",
+    "FAQ_CUSTOMIZE_DESIGN": "Can I customize the design?",
+    "FAQ_CUSTOMIZE_DESIGN_ANSWER": "Yes! Ever Works is built with Tailwind CSS and offers extensive customization options.",
+    "FAQ_HOSTING": "What hosting do you recommend?",
+    "FAQ_HOSTING_ANSWER": "We recommend Vercel for the best performance, but Ever Works works with any Node.js hosting provider.",
+    "HERO_SPECIAL_OFFER": "Special Gift: 50% off",
+    "HERO_CREATE_YOUR": "Transform your",
+    "HERO_DIRECTORY": "ideas",
+    "HERO_IN_MINUTES": "in minutes",
+    "HERO_SUBTITLE": "Revolutionary web development foundation, built with",
```

Biome (1.9.4)

[error] The key HERO_SUBTITLE was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)